src/renderer/components/item-object-list/content.tsx

```diff
@@ -202,7 +180,7 @@ export class ItemListLayoutContent<I extends ItemObject> extends React.Component
       return null;
     }
 
-    return this.props.getItems().map(item => this.getRow(item.getId()));
+    return this.props.getItems().map(item => this.getRow(item.getId(), item));
```

Comment:
Why two separate parameters? `item` should be enough.

Reply:
Unfortunately not because `getRow` must be match the `(id: string) => React.ReactNode` signature (at least I thought it did).

Reply:
Yes, it all goes from the legacy part. Virtual list rows will use following notation for ` getRow` prop:
```
getRow?: (uid: string | number, item?: ItemObject) => React.ReactElement<TableRowProps>;
```

All existing extensions using virtual tables are waiting a `uid` arguments for their internal `getRow` functions. But obviously passing an item itself is much faster than later looking for it in the `items` array. So I decided to use second optional argument for that.